_episodes/03-sql-joins.md

```diff
@@ -152,12 +180,37 @@ could do something like
 
 > ## Challenge:
 >
-> - Write a query that returns the number of genus of the animals caught in each plot in descending order.
+> - Write a query that returns the number of animals caught per genus in each plot. 
```

Comment:
"in each genus" might be clearer than "per genus" (which could be read as the number of animals divided by the number of genera).